modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/document/library/DocumentLibraryDDMFormFieldTemplateContextContributor.java

```diff
@@ -203,6 +203,31 @@ protected String getFileEntryURL(
 		return html.escape(sb.toString());
 	}
 
+	protected String getGuestUploadURL(
+		DDMFormFieldRenderingContext ddmFormFieldRenderingContext,
+		long folderId, HttpServletRequest httpServletRequest) {
```

Comment:
@carolmariaabb, I think it would be better to add the `folderId` as the first parameter once it's a `id` and considering the previous reviews of Brian, what do you think?

Reply:
Answered above ⬆️